src/main/java/pages/ContactUsPage.java

```diff
@@ -6,22 +6,78 @@
 import org.openqa.selenium.support.ui.ExpectedConditions;
 import org.openqa.selenium.support.ui.WebDriverWait;
 
-public class ContactUsPage {
 
-    private WebDriver driver;
-    private int timeOut = 5;
+public class ContactUsPage extends Page{
 
     private By contactUsPageTitle = By.xpath("//h1[contains(text(),'Contact')]");
+    private By contactUsFirstNameInputField = By.cssSelector("input[name='firstName']");
+    private By contactUsLastNameInputField = By.cssSelector("input[name='lastName']");
+    private By contactUsEmailInputField = By.cssSelector("input[name='email']");
+    private By contactUsCompanyNameInputField = By.cssSelector("input[name='companyName']");
+    private By contactUsPhoneNumberInputField = By.cssSelector("input[name='homePhone']");
+    private By contactUsMessageInputField = By.cssSelector("textarea[name='message']");
+    private By submitButton = By.cssSelector(".block-button.block-button--primary");
+//    private By firstName
 
     public ContactUsPage(WebDriver driver){
+        super(driver);
         PageFactory.initElements(driver,this);
-        this.driver = driver;
     }
 
     public Boolean isTitleVisible(){
-        WebDriverWait wait = new WebDriverWait(driver,timeOut);
         wait.until(ExpectedConditions.presenceOfElementLocated(contactUsPageTitle));
         return driver.findElement(contactUsPageTitle).isDisplayed();
     }
+//
+    public Boolean isFirstNameInputFieldVisible() {
+        wait.until(ExpectedConditions.presenceOfElementLocated(contactUsFirstNameInputField));
+        return driver.findElement(contactUsFirstNameInputField).isDisplayed();
+    }
 
+    public Boolean isSubmitButtonVisible(){
+        wait.until(ExpectedConditions.presenceOfElementLocated(submitButton));
+        return driver.findElement(submitButton).isDisplayed();
+    }
+    public void fillInFirstName(String firstName){
+        driver.findElement(contactUsFirstNameInputField).click();
+        driver.findElement(contactUsFirstNameInputField).sendKeys(firstName);
+    }
+    public void fillInLastName(String lastName){
+        driver.findElement(contactUsLastNameInputField).click();
+        driver.findElement(contactUsLastNameInputField).sendKeys(lastName);
+    }
+    public void fillInEmail(String email){
```

Comment:
Ik zie wel heel veel code duplicatie in deze methodes. Zou je hier niet een betere oplossing voor kunnen bedenken?